profile/index.php

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
You should get some php above this DOCTYPE to check if the user is logged in, and if they aren't then redirect them to the unauthorized page. See management/appointments/index.php. That one does a permissions check, you'll instead do a logged in check:
```
if(!$USER->isLoggedIn()) { 
// redirect stuff
}
```